gettsim/transfers/rente.py

```diff
@@ -274,25 +300,419 @@ def ges_rente_zugangsfaktor(
     return out
 
 
-def ges_rente_regelaltersgrenze(geburtsjahr: int, ges_rente_params: dict) -> float:
-    """Calculates the age, at which a worker is eligible to claim his full pension.
+def ges_rente_regelaltersgrenze(
+    geburtsjahr: int, ges_rente_vorraus_regelrente: bool, ges_rente_params: dict
+) -> float:
+    """Calculates the age, at which a person is eligible to claim the regular pension.
+        Normal retirement age (NRA). This pension cannot be claimed earlier than at the
+        NRA, ie it does not serve as reference for calculating deductions. However, it
+        serves as reference for calculating gains in the Zugangsfakor in case
+        of later retirement.
+
+    Parameters
+    ----------
+    geburtsjahr
+        See basic input variable :ref:`geburtsjahr <geburtsjahr>`.
+    ges_rente_params
+        See params documentation :ref:`ges_rente_params <ges_rente_params>`.
+    ges_rente_vorraus_regelrente
+        See :func: ``ges_rente_vorraus_regelrente``.
+
+    Returns
+    -------
+    """
+    if ges_rente_vorraus_regelrente:
```

Comment:
I think we could also check `ges_rente_vorraus_regelrente` in the function `ges_rente_zugangsfaktor` and define the actual `ges_rente_regelaltersgrenze` for all individuals. That way we would not need the ugly `out = 9000`.

Reply:
I moved the eligibility checks to '''ges_rente_grenz_voll_altersrente''' and '''referenz_alter_abschlag''' and got rid of the '''out = 9000'''. I did not shift it to '''ges_rente_zugangsfaktor''' to not overload the function there.